### PR TITLE
Pr fixes v1 1

### PR DESCRIPTION
**Pull request by atra94**
Corrections to the comments to the nightly -> master PR for GEM v1.1.0. 

Cannot push directly to the nightly branch. Therefore, I needed a new branch + PR. 

The two points regarding the name of the StateActionProcessor class and their description in the environments docstrings are still open.

### CODE REVIEW

File: gym_electric_motor/physical_system_wrappers/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from .state_action_processor import StateActionProcessor
+from .physical_system_wrapper import PhysicalSystemWrapper
```
Comment (max-schenke) [inline gym_electric_motor/physical_system_wrappers/__init__.py:1]: physical_system_wrapper or _wrappers?

**Review by max-schenke — COMMENTED** [review]
One minor remark that I am just unsure about. 
Thanks a lot for the renaming, we should be able to merge this soon.

File: gym_electric_motor/physical_system_wrappers/__init__.py (hunk 1 above)
Comment (atra94) [reply gym_electric_motor/physical_system_wrappers/__init__.py:1]: wrapper is correct. The file containing the base PhysicalSystemWrapper is called physical_system_wrapper.py. The folder containing all wrappers is named physical_system_wrappers.

**Review by max-schenke — APPROVED** [review]
Case distinction for EESM looks good to me.

**Pull request merged**